CHANGELOG_PENDING.md

```diff
@@ -17,7 +17,8 @@
 ### FEATURES:
 
 ### IMPROVEMENTS:
-- [p2p] [\#3463](https://github.com/tendermint/tendermint/pull/3463) Do not log "Can't add peer's address to addrbook" error for a private peer
+- [rpc] \#3534 Add support for batched requests/responses in JSON RPC
+- [p2p] \#3463 Do not log "Can't add peer's address to addrbook" error for a private peer
```

Comment:
please return the URL. I doubt that our script can expand links to PRs (note the link refers to a PR, not an issue)

Reply:
It can because github redirects to prs; e.g., try this `https://github.com/tendermint/tendermint/issue/3534`

Reply:
Ah, fixing it now 😁

Reply:
Thought I'd just fix it anyways - as long as it doesn't cause the scripts to try to double-link it somehow (i.e. insert a link to a non-existent issue within the PR link). Haven't taken a look at how those scripts work yet.

Reply:
Weren't we supposed to only include links here to PRs though? I thought that's what we decided from one of the discussions earlier in the year? Unless I got them confused.

Reply:
> Haven't taken a look at how those scripts work yet.

https://github.com/tendermint/tendermint/blob/75ffa2bf1c7d5805460d941a75112e6a0a38c039/scripts/linkify_changelog.py#L4-L13

You can link bot, PRs, and issues (due to github redirecting).

> Weren't we supposed to only include links here to PRs though?

@ebuchman often links PRs. Everyone else links only issues. We haven't really decided a strict policy here. I think whatever makes more sense in the particular context is fine 👍